content/ch-appendix/linear_algebra.ipynb

```diff
@@ -4454,7 +4454,7 @@
     "\n",
     "<br>\n",
     "\n",
-    "The reason unitary matrices are important will become more apparent in the section on Hilbert spaces, and more so in the quantum mechanics subtopic of this textbook.  The basic idea is that evolution of a quantum state by application of a unitary matrix \"preserves\" the quantum state.\n",
+    "The reason unitary matrices are important will become more apparent in the section on Hilbert spaces, and more so in the quantum mechanics subtopic of this textbook.  The basic idea is that evolution of a quantum state by application of a unitary matrix \"preserves\" the magnitude or length of the quantum state.\n",
```

Comment:
Is there a double space after the first sentence?

As for the word itself, perhaps we could go with "norm" rather than length, since the latter can be confusing (length can also be interpreted as number off possible measured states, number if qubits etc)

Reply:
yes, I agree, the norm would the better word